### PR TITLE
 [EDA] Relationship mappings new/edit/delete

### PR DESCRIPTION
**Pull request by iskander-sfdc**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/classes/RelationshipLookupMapper_TEST.cls [hunk 1]
```diff
@@ -25,68 +25,273 @@
...
+            Name = 'test' + system.now(),
+            Active__c = true,
+            Female__c = 'wife',
+            Male__c = 'husband',
+            Neutral__c = 'spouse'
+        );
 
         Test.startTest();
-             List<RelationshipLookupModel> relationshipLookupModelsList = RelationshipLookupMapper.getInstance().getRelationshipLookupModels();
+        List<RelationshipLookupModel> relationshipLookupModelsList = RelationshipLookupMapper.getInstance()
```
Comment (MatthewBlanski) [inline force-app/main/default/classes/RelationshipLookupMapper_TEST.cls:100]: This is not a required change and something I've been commenting very loosely upon. Consider what you are testing uniquely within the test block and what metrics you are looking to retrieve? Is getting the instance part of those metrics or are we testing the performance of an individual method?

File: force-app/main/default/lwc/relationshipMappingModalBody/relationshipMappingModalBody.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                        type="toggle"
+                        label={labelReference.isActive}
+                        id={inputAttributeReference.isActive}
+                        name={inputAttributeReference.isActive}
+                        onchange={handleIsActiveChange}
+                        checked={isActive}
+                        value={isActive}
+                    ></lightning-input>
+                </div>
+            </template>
```
Comment (MatthewBlanski) [inline force-app/main/default/lwc/relationshipMappingModalBody/relationshipMappingModalBody.html:80]: Do we need to reassert the same condition across all of these? I would imagine we can wrap them in a single template if:true. This is functional, but causes more checks than are necessary on the render.

**Review by MatthewBlanski — CHANGES_REQUESTED** [review]
I left two simple comments, however, the navigation label for this item should be stgTabReciprocalSettings in the nav menu. Right now it is "New Reciprocal Settings". Please change that over in the eda settings navigation component.

**Review by xinnanlisf — APPROVED** [review]

File: force-app/main/default/lwc/relationshipMappingModalBody/relationshipMappingModalBody.html (hunk 2 above)
Comment (iskander-sfdc) [reply force-app/main/default/lwc/relationshipMappingModalBody/relationshipMappingModalBody.html]: Thanks, I changed the template to use only one if:true condition

**Comment by iskander-sfdc** [comment]
> I left two simple comments, however, the navigation label for this item should be stgTabReciprocalSettings in the nav menu. Right now it is "New Reciprocal Settings". Please change that over in the eda settings navigation component.

Changed the label in the nav menu to stgTabReciprocalSettings

File: force-app/main/default/classes/RelationshipLookupMapper_TEST.cls (hunk 1 above)
Comment (iskander-sfdc) [reply force-app/main/default/classes/RelationshipLookupMapper_TEST.cls:100]: I think this is just a formatting change in this specific place

**Review by MatthewBlanski — APPROVED** [review]

**Comment by rhakeshj** [comment]
QA passed

**Pull request merged**